integration/core/test_cli.py

```diff
@@ -592,17 +593,19 @@ def test_backup_core(bin, controller_client, replica_client,
     ])
     assert v.replicaCount == 2
 
+    env = dict(os.environ)
+    o = urlparse(backupTarget)
```

Comment:
The only thing you used here is `o.scheme`. Just do `backupType = urlparse(backupTarget).scheme`.